### ISSUE TITLE
SnapshotVR

### ISSUE THREAD

**Issue by Maikel1990** [CubeCoders/AMPTemplates#617]
# Module Request

Note: Please try setting up a configuration yourself before raising an issue to request a configuration:  ~~https://config.getamp.sh/~~
***There is a newer beta version available at https://iceofwraith.github.io/GenericConfigGen/

## Game Information:

Game Title: SnapshotVR
How to obtain: https://brief-cornflower-f79.notion.site/Snapshot-VR-Docs-01b806512e794de8b6cafdc4199bcd46
Suppoted OSs: Windows and Linux
Supports RCON/Console input? Not sure about it

## I confirm:

- [x] that I have searched for an existing module request for this application.
- [x] that the server is an official server distributed by the games publishers/developers.
- [x] that the server is available to the general public.
- [x] that the server can be run headless (without a GUI).
- [x] I have already attempted to create a configuration myself using the configuration tool

**Comment by Greelan** [comment]
Template in `Greelan/AMPTemplates:dev`

How do you actually connect to the server?

**Comment by Maikel1990** [comment]
How do you mean? You connect to servers with the in-game server list.

**Comment by Greelan** [comment]
Right. The server doesn't open a fixed port so I'd assumed it used a relay network and you'd couldn't direct connect

**Comment by Maikel1990** [comment]
From what I've read in the Discord server is that every client gets their own port.

"Snapshot uses ephemeral ports to connect, so there's not a set of "allow these ports" instructions. This is specific to your router and your internet connection."

Thanks for helping me out. Will test soon if everything works.

**Comment by Greelan** [comment]
That's a sorta misleading/confusing comment. There are no ports to forward for this server so it just uses a relay network

**Comment by Maikel1990** [comment]
I searched for relay on their discord and you are right.

"It's Photon Bolt backend, not PUN. Primary difference being that Photon Cloud is still used for "matchmaking" to the target server and as a backup server for relay if P2P fails, but that most traffic will be P2P after connect"
